arbnode/dataposter/data_poster.go

```diff
@@ -599,6 +601,7 @@ var DefaultDataPosterConfig = DataPosterConfig{
 	UrgencyGwei:            2.,
 	MaxMempoolTransactions: 64,
 	MinTipCapGwei:          0.05,
+	MaxTipCapGwei:          0.1,
```

Comment:
Let's increase the default config value to 5 gwei. I think that's large enough to let us get in the tx quickly during congestion, but also not too large that we can't continue escalating the tip safely with replace-by-fee.

Reply:
Done.